Jenkinsfile_k8s

```diff
@@ -70,7 +70,12 @@ spec:
   restartPolicy: "Never"
   containers:
     - name: img
-      image: r.j3ss.co/img
+      image: r.j3ss.co/img:v0.5.11
+      command:
+      - cat
+      tty: true
+    - name: cst
+      image: gcr.io/gcp-runtimes/container-structure-test:v1.9.1
       command:
       - cat
```

Comment:
the container-structure-test image doesn't appear to have a shell, so `cat` doesn't exist

Reply:
`zemanlx/container-structure-test:v1.9.1-alpine` might be a better option?